internal/pwru/types.go

```diff
@@ -48,6 +50,7 @@ func (f *Flags) SetFlags() {
 	flag.BoolVar(&f.OutputSkb, "output-skb", false, "print skb")
 	flag.BoolVar(&f.OutputStack, "output-stack", false, "print stack")
 	flag.Uint64Var(&f.OutputLimitLines, "output-limit-lines", 0, "exit the program after the number of events has been received/printed")
+	flag.Uint32Var(&f.PerCPUBuffer, "per-cpu-buffer", 0, "per CPU buffer in bytes")
```

Comment:
Nit: instead of the default `0`, you could use `os.Getpagesize()`. This would also allow us to get rid of the if-statement at the line 104 in `.main.go`

Reply:
Excellent point - will do